### PR TITLE
[KEYCLOAK-3482]

### PR DESCRIPTION
**Pull request by abstractj**

### CODE REVIEW

File: server_admin/topics/threat/admin.adoc [hunk 1]
```diff
@@ -93,4 +93,29 @@ Equivalent configuration using CLI commands:
...
 /subsystem=undertow/configuration=filter/expression-filter=portAccess:add(,expression="path-prefix('/auth/admin') and not equals(%p, 8444) -> response-code(403)")
 /subsystem=undertow/server=default-server/host=default-host/filter-ref=portAccess:add()
-----
\ No newline at end of file
+----
+
+==== Server version
+
+There are scenarios where people may want to hide the {project_name} version. It is possible to create Undertow filters redirecting all the requests to `/auth/version` to any page you want, preventing people to know the details about the server. The following example do not permit access to `/auth/version`:
+
```
Comment (matthewhelmke) [inline server_admin/topics/threat/admin.adoc]: Edit suggestion:
There are scenarios where it is desirable to hide the {project_name} version. To do so, create Undertow filters redirecting all the requests to `/auth/version` to any page you want, which will then prevent access to the details about the server. The following example prevents access to `/auth/version`:

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
Thank you for doing this! I've edited the paragraph, so it should be a quick copy/paste update on your end.

**Comment by abstractj** [comment]
@matthewhelmke thank you, updated.

**Review by matthewhelmke — APPROVED** [review]

**Comment by stianst** [comment]
@abstractj I would just remove the endpoint in 7.3, but document for 7.2

**Comment by abstractj** [comment]
Closing it and will do what Stian suggested.

**Pull request closed without merging**